src/cli.ts

```diff
@@ -264,7 +275,17 @@ populatePatientBundles().then(async patientBundles => {
       });
     }
 
-    console.log(JSON.stringify(result?.results, null, 2));
+    if (program.outFile) {
```

Comment:
Nitpicking, but I would change this nested if to just an 
```
if(program.outFile === true){...}
else if(program.outFile){...}
else{...}
```
with a couple comments to clear up what the conditions represent